app/services/reporting/total_user_count_report.rb

```diff
@@ -15,6 +15,13 @@ def total_user_count_report
       ]
     end
 
+    def total_verified_users_report
+      [
+        ['Total verified users'],
+        [verified_user_count],
+      ]
+    end
```

Comment:
Aside: I like the pattern of keeping this because I think it makes the `...emailable_report` method tidier, and separates the report-data logic from the formatting-metadata "logic." _But_, this `_report` method is really now kind of a legacy pattern. I propose keeping it for now but it could become private or something. 🤔